pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -130,6 +130,11 @@ func (r *ReconcileStorageCluster) ensureCephObjectStores(instance *ocsv1.Storage
                 r.reqLogger.Info("not creating a CephObjectStore because the platform is GCP")
                 return nil
         }
+	if platform == PlatformAzure {
```

Comment:
Instead, can we just check if platform is one of the supported platforms and skip CephObjectStore creation?